### DISCUSSION TITLE
[New Feature] Move Super Weapons Camoes To Tactical View

### DISCUSSION THREAD

**Discussion by Onidotmoe** [Vinifera-Developers/Vinifera#1340] [Feature Requests]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, videos, links, etc)*.

### Description

In C&C3 Super Weapons and consumable powers are not in the sidebar but instead on the right side of the screen.

This would be neat, adding a option to toggle it would be fine too.

### Proposed Code (optional)
```
; in rules.ini
[MultiSpecial]
; Options to control where it is displayed
DisplayTacticalbar=yes
DisplaySidebar=no
; Vanilla has this key
SidebarImage=MltiIcon
; Adding this too would probably make sense, if not set, use SidebarImage
TacticalbarImage=somethingelse

```

### Additional Context

![image](https://user-images.githubusercontent.com/32226493/184448886-1ca7a036-076a-42e8-8249-9c6a8853b487.png)

**Comment by DuisPlay** [comment]
A Very Festive Scarecrow (creator of Tiberian Revenge) made a package for RA2 + Phobos. (can be found on Mod Heaven Discord) 

[TSSWSB.zip](https://github.com/user-attachments/files/22307609/TSSWSB.zip)

<img width="1600" height="900" alt="Image" src="https://github.com/user-attachments/assets/188fba00-aba7-4f86-8cd7-83d25d874fac" />
